VEHICLE/CopyVehicleDamages.md

```diff
@@ -0,0 +1,19 @@
+---
+ns: VEHICLE
+aliases: ["0xE44A982368A4AF23"]
+---
+## COPY_VEHICLE_DAMAGES
+
+```c
+// 0xE44A982368A4AF23 0xBAE491C7
+void COPY_VEHICLE_DAMAGES(Vehicle sourceVehicle, Vehicle targetVehicle);
+```
+
+```
+Copy the damage like broken bumpers and broken lights from a vehicle and paste then on another vehicle.
```

Comment:
"...and paste then on another vehicle." - Tiny grammar mistake. Should be `them`